tests/test_repository_contract.py

```
@@ -303,14 +430,111 @@ def test_terminal_external_action_schema_requires_signed_resolution_proof(self):
+    def test_rc1_uses_patched_cryptography_line(self):
         pyproject = (ROOT / "pyproject.toml").read_text(encoding="utf-8")
         lock = (ROOT / "requirements-governance.lock").read_text(encoding="utf-8")
-        self.assertIn('version = "0.2.0.dev8"', pyproject)
+        self.assertIn('version = "0.2.0rc1"', pyproject)
         self.assertIn('cryptography>=50,<51', pyproject)
         self.assertIn("cryptography==50.0.0", lock)
         self.assertNotIn("cryptography==46.0.7", lock)
 
+    def test_release_workflow_is_manual_isolated_and_attested(self):
+        source = (ROOT / ".github/workflows/publish.yml").read_text(encoding="utf-8")
+        workflow = yaml.load(source, Loader=yaml.BaseLoader)
```

Ruff (0.16.1)

[error] Probable use of unsafe loader `BaseLoader` with `yaml.load`. Allows instantiation of arbitrary objects. Consider `yaml.safe_load`.

(S506)